src/main/java/github/scarsz/discordsrv/objects/threads/NicknameUpdater.java

```diff
@@ -117,14 +120,20 @@ public void setNickname(Member member, OfflinePlayer offlinePlayer) {
 
             nickname = DiscordSRV.config().getString("NicknameSynchronizationFormat")
                     .replace("%displayname%", player.getDisplayName() != null ? player.getDisplayName() : player.getName())
-                    .replace("%username%", player.getName());
+                    .replace("%username%", player.getName())
+                    .replace("%discord_name%", member.getUser().getName())
+                    .replace("%discord_discriminator%", member.getUser().getDiscriminator());
 
             nickname = PlaceholderUtil.replacePlaceholders(nickname, player);
         } else {
             nickname = offlinePlayer.getName();
         }
 
         nickname = MessageUtil.strip(nickname);
+        if (nickname.length() > 32) {
+            DiscordSRV.error("The new nickname for " + offlinePlayer.getName() + " is too long, reducing it to 32 characters.");
```

Comment:
As I already stated on Discord, would like to see someway to avoid this showing up as a error, incase the user is ok with them being shortened